src/xmipp/libraries/data/polar.h

```diff
@@ -586,17 +586,16 @@ class Polar
         // Add additional points on the inside and outside of the rings
         // Add a maximum of "extra_shell" rings
         // Set data to zero here
-        double first_ring  = ring_radius[0];
-        double last_ring   = ring_radius[rings.size()-1];
-        float outer       = last_ring + extra_shell;
-        float inner       = XMIPP_MAX(0.,first_ring - extra_shell);
-        
-        // for (float radius = 0.; radius < outer; radius +=1.)
-        for (int radius = 0; radius < outer; radius++)
+        auto first_ring  = (int)floor(ring_radius[0]);
```

Comment:
why do you add `floor` / `ceil`?

Reply:
It is not my change, but I suppose that he wants the rings to be numbered